# HG changeset patch
# User Rik <rik@octave.org>
# Date 1543621587 28800
#      Fri Nov 30 15:46:27 2018 -0800
# Node ID 7bd67e786e5fd656137755e66ffa59fa9a33515f
# Parent  b68c41cb0f995e2de89b9986198fefdc44e3b9db
decic.m: Fix failing BIST test.

* decic.m: Use "FinDiffType"="central" in options to fminunc.  Remove "display"
property from list of options passed to fminunc.

diff --git a/scripts/ode/decic.m b/scripts/ode/decic.m
--- a/scripts/ode/decic.m
+++ b/scripts/ode/decic.m
@@ -157,17 +157,17 @@ function [y0_new, yp0_new, resnorm] = de
                "decic: RelTol must be a scalar value");
       else
         TolX = options.RelTol;
       endif
     endif
   endif
 
   x0 = [y0(! fixed_y0); yp0(! fixed_yp0)];
-  opt = optimset ("tolfun", TolFun, "tolx", TolX, "display", "iter-detailed");
+  opt = optimset ("tolfun", TolFun, "tolx", TolX, "FinDiffType", "central");
   x = ...
     fminunc (@(x) objective (x, t0, y0, fixed_y0, yp0, fixed_yp0, nl, nu, fun),
              x0, opt);
 
   y0_new  = y0;
   yp0_new = yp0;
 
   y0_new(! fixed_y0)   = x(1:nl);
